app/controllers/assignments_controller.rb

```diff
@@ -717,16 +678,21 @@ def build_hash_from_zip(zip_file, hash_to_build)
   # Builds an uploaded assignment/peer review assignment from its properties file
   # Precondition: prop_file must be a Zip::Entry object
   #               If +parent_assignment+ is not nil, this is a peer review assignment.
```

Comment:
You should update this comment to reflect the parameter change (and new key `:parent_assessment_short_identifier`)